packages/wrangler/src/index.tsx

```diff
@@ -537,11 +537,11 @@ export function createCLIParser(argv: string[]) {
 			);
 
 			const scriptTag = scriptMetadata.default_environment.script.tag;
-			const deployments = await fetchResult<DeploymentListRes>(
-				`/accounts/${accountId}/workers/versions/by-script/${scriptTag}`
+			const { deployments } = await fetchResult<DeploymentListRes>(
+				`/accounts/${accountId}/workers/deployments/by-script/${scriptTag}`
```

Comment:
The endpoint itself might take a while to change, keep using `.../workers/versions/...` for now.